src/cluster/serializer/binary-serialization-format/Serializer.cc

```diff
@@ -0,0 +1,191 @@
+#include "zeek/cluster/serializer/binary-serialization-format/Serializer.h"
+
+#include <cstdint>
+
+#include "zeek/ID.h"
+#include "zeek/Reporter.h"
+#include "zeek/SerializationFormat.h"
+#include "zeek/Type.h"
+#include "zeek/Val.h"
+#include "zeek/cluster/Serializer.h"
+#include "zeek/logging/Types.h"
+#include "zeek/threading/SerialTypes.h"
+
+using namespace zeek::cluster;
+
+// #define SERIALIZER_DEBUG(...) fprintf(stderr, __VA_ARGS__)
```

Comment:
I've switched this over to use the following so that `zeek -B plugin-Zeek-Binary_Serializer` enables the messages instead.
```
#define SERIALIZER_DEBUG(...) PLUGIN_DBG_LOG(zeek::plugin::Zeek_Binary_Serializer::plugin, __VA_ARGS__)
```

Thanks for poking.